docs/content/en/main/concepts/syncer.md

```diff
@@ -96,14 +96,40 @@ You can then create foo.bar in this workspace, or create an APIBinding in anothe
 To sync resource from another existing APIExport in the KCP server, run
 
 ```sh
-kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace|another-apiexport -o syncer.yaml
+kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
```

Comment:
```suggestion
kubectl kcp workload sync <my-workspace> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
```

Do we want to consistently refer to logical clusters as workspaces?

Reply:
> Do we want to consistently refer to logical clusters as workspaces?

I think we should yes

Reply:
why not put `<mysynctarget>` then ?

Reply:
```suggestion
kubectl kcp workload sync <my-synctarget> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
```